### PR TITLE
MP-4

### LINKED ISSUE #6: MP-4 User entity
- Создать сущности User и UserProfile с [полями](https://github.com/varya-kot/mambet-pages/wiki/DB-Structure-‐-Личный-кабинет)
- Написать миграции Liquibase для базы PostgreSQL
- Создать репозитории Hibernate для этих сущностей

### PR DESCRIPTION
**Pull request by ivamijava**
Added entities and connected database
Closes #6

### CODE REVIEW

File: src/main/java/ru/mambetpages/entity/User.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+public class User implements Serializable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    @Column(name = "id")
+    private UUID id;
+
+    @Column(name = "email", unique = true, nullable = false)
+    private String email;
+
+    @Column(name = "hashedPassword", nullable = false)
```
Comment (VaryaGet) [inline src/main/java/ru/mambetpages/entity/User.java]: Rename all needed columns in snake case

File: src/main/java/ru/mambetpages/repository/UserProfileRepository.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package ru.mambetpages.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import ru.mambetpages.entity.UserProfile;
+
+import java.util.UUID;
+
```
Comment (VaryaGet) [inline src/main/java/ru/mambetpages/repository/UserProfileRepository.java:8]: @Repository

File: src/main/java/ru/mambetpages/repository/UserRepository.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package ru.mambetpages.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import ru.mambetpages.entity.User;
+
+import java.util.UUID;
+
```
Comment (VaryaGet) [inline src/main/java/ru/mambetpages/repository/UserRepository.java:8]: @Repository

**Review by VaryaGet — CHANGES_REQUESTED** [review]

File: src/main/java/ru/mambetpages/entity/User.java (hunk 1 above)
Comment (ivamijava) [reply src/main/java/ru/mambetpages/entity/User.java]: Done!

File: src/main/java/ru/mambetpages/repository/UserProfileRepository.java (hunk 2 above)
Comment (ivamijava) [reply src/main/java/ru/mambetpages/repository/UserProfileRepository.java:8]: Done!

File: src/main/java/ru/mambetpages/repository/UserRepository.java (hunk 3 above)
Comment (ivamijava) [reply src/main/java/ru/mambetpages/repository/UserRepository.java:8]: Done!

File: src/main/java/ru/mambetpages/repository/UserProfileRepository.java (hunk 2 above)
Comment (ArtemGet) [reply src/main/java/ru/mambetpages/repository/UserProfileRepository.java:8]: Redundant annotation, remove it

File: src/main/java/ru/mambetpages/repository/UserRepository.java (hunk 3 above)
Comment (ArtemGet) [reply src/main/java/ru/mambetpages/repository/UserRepository.java:8]: Redundant annotation, remove it

**Review by VaryaGet — APPROVED** [review]

File: src/main/resources/application.yaml [hunk 4]
```diff
@@ -2,16 +2,39 @@ spring:
...
+    host: ${MAIL_HOST:smtp.yandex.ru}
+    port: ${MAIL_PORT:465}
+    username: ${MAIL_USERNAME:email_username}
+    password: ${MAIL_PASSWORD:email_password}
+    email_alias: ${EMAIL_ALIAS:email_alias}
+    protocol: ${MAIL_PROTOCOL:smtps}
+
+#DB settings
+  datasource:
+    driver-class-name: org.postgresql.Driver
```
Comment (ArtemGet) [inline src/main/resources/application.yaml:21]: Use parameters like ${PG_USERNAME:}

File: src/main/resources/application.yaml [hunk 5]
```diff
@@ -2,16 +2,39 @@ spring:
...
+    username: username
+    password: password
+    url: jdbc:postgresql://localhost:5432/mambet_pages
+  jpa:
+    hibernate :
+      ddl-auto : none
+    show-sql: true
```
Comment (ArtemGet) [inline src/main/resources/application.yaml]: default false

**Review by ArtemGet — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yaml (hunk 4 above)
Comment (ivamijava) [reply src/main/resources/application.yaml:21]: Done!

File: src/main/resources/application.yaml (hunk 5 above)
Comment (ivamijava) [reply src/main/resources/application.yaml]: Done!

**Review by ArtemGet — APPROVED** [review]

**Pull request merged**